stylix/testbed.nix

```diff
@@ -146,23 +146,42 @@ let
           ]
       );
 
-      system = lib.nixosSystem {
-        inherit (pkgs) system;
-
-        modules = [
-          commonModule
-          applicationModule
-          inputs.self.nixosModules.stylix
-          inputs.home-manager.nixosModules.home-manager
-          testbed.path
-
-          {
-            inherit stylix;
-            system.name = name;
-          }
-        ];
+      systems = {
+        clean = lib.nixosSystem {
+          inherit (pkgs) system;
+
+          modules = [
+            commonModule
+            applicationModule
+            inputs.home-manager.nixosModules.home-manager
+            testbed.path
+            { system.name = name; }
+          ];
+        };
+
+        disabled = systems.clean.extendModules {
+          modules = [
+            inputs.self.nixosModules.stylix
+          ];
+        };
+
+        enabled = systems.disabled.extendModules {
+          modules = [
+            { inherit stylix; }
+          ];
+        };
```

Comment:
What about removing the `enable = true;` attribute from:

https://github.com/danth/stylix/blob/2fb8321ea16c595e0208b22021ddaf1f471c634a/stylix/testbed.nix#L206-L241

Then, we define the `disabled` variant with `enable = false;` and the `enabled` variant with `enable = true;`?

Reply:
The churrent implementation should also fail if there are any "option _ must be set" errors when having no Stylix configuration- as we want all features to be optional this is another useful check.

Perhaps this should be documented in a comment.